pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -977,7 +977,17 @@ type ClusterConfig struct {
 	//     - yes
 	//     - false
 	//     - no
+	//
+	// Deprecated: Use `AllowSchedulingOnControlPlanes` instead.
```

Comment:
we can instead drop the whole docblock here and replace it with:

```
	// docgen:nodoc
```

this way it will disappear from 1.2 docs altogether, which imho makes sense